Validation/RecoTau/interface/TauValidationMiniAOD.h

```diff
@@ -13,8 +13,11 @@
 
 */
 // Original Author: Aniello Spiezia On August 13, 2019
-// user include files
+// Updated April, 2020 by Ece Asilar and Gage DeZoort
+// Updated July, 2023 by Gourab Saha
 
+// user include files
+//#include "FWCore/Framework/interface/EDAnalyzer.h"
 #include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
```suggestion
```
Not needed here, the plugin is defined in Validation/RecoTau/src/SealModule.cc

Reply:
Removed